.github/workflows/test-and-lint.yml

```diff
@@ -12,17 +12,14 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
-    - name: Setup MiniConda
+    - name: Install Conda Deps
```

Comment:
changes reviewed in [PR to this branch](https://github.com/simularium/simulariumio/pull/95)